src/components/calcite-pick-list/calcite-pick-list.tsx

```diff
@@ -11,7 +11,8 @@ import {
   h
 } from "@stencil/core";
 import { ICON_TYPES, TEXT } from "./resources";
-import { VNode } from "@esri/calcite-components/dist/types/stencil.core";
+import sharedListMethods from "./shared-list-logic";
```

Comment:
nitpick: might be nicer to import the methods directly as needed instead of having to call `sharedListMethods.method` every time they are used.

`import {method} from "..."`

Reply:
Would you be ok if I just added a destructuring call after the import for now?

Reply:
👍 Its just a nitpick.

Reply:
added destructuring statement. Avoiding splitting into multiple exports for now since it doesn't help with tree-shaking (they are all used).